diff -Nru logback-1.1.5/logback-access/pom.xml logback-1.1.5.jetty/logback-access/pom.xml
--- logback-1.1.5/logback-access/pom.xml	2016-02-13 23:24:13.000000000 +0100
+++ logback-1.1.5.jetty/logback-access/pom.xml	2016-02-29 15:10:40.482323064 +0100
@@ -46,6 +46,12 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
+      <scope>compile</scope>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
       <groupId>org.codehaus.janino</groupId>
       <artifactId>janino</artifactId>
       <scope>compile</scope>
diff -Nru logback-1.1.5/logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java logback-1.1.5.jetty/logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java
--- logback-1.1.5/logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java	2016-02-12 22:52:07.000000000 +0100
+++ logback-1.1.5.jetty/logback-access/src/main/java/ch/qos/logback/access/jetty/RequestLogImpl.java	2016-02-29 15:10:40.482323064 +0100
@@ -214,11 +214,6 @@
     started = false;
   }
 
-  @Override
-  public boolean isRunning() {
-    return started;
-  }
-
   public void setFileName(String fileName) {
     this.fileName = fileName;
   }
@@ -232,26 +227,6 @@
     return started;
   }
 
-  @Override
-  public boolean isStarting() {
-    return false;
-  }
-
-  @Override
-  public boolean isStopping() {
-    return false;
-  }
-
-  @Override
-  public boolean isStopped() {
-    return !started;
-  }
-
-  @Override
-  public boolean isFailed() {
-    return false;
-  }
-
   public boolean isQuiet() {
     return quiet;
   }
@@ -315,14 +290,7 @@
     return fai.getFilterChainDecision(event);
   }
 
-  @Override
-  public void addLifeCycleListener(Listener listener) {
+  public void log(Request request, int status, long written) {
     // we'll implement this when asked
   }
-
-  @Override
-  public void removeLifeCycleListener(Listener listener) {
-    // we'll implement this when asked
-  }
-
 }
